tests/ui/pages/desktop/base.py

```diff
@@ -26,6 +26,15 @@ def login(self, email, password):
     def logout(self):
         self.header.click_logout()
 
+    def wait_for_page_to_load(self):
+        body = self.find_element(By.TAG_NAME, 'body')
+        self.wait.until(lambda s: 'firefox' in body.get_attribute('class'))
+
+    @property
+    def wait_for_page(self):
+        url = self.selenium.current_url
+        self.wait.until(lambda _: self.selenium.current_url != url)
```

Comment:
Using my method against a deployed instance doesnt work. The devs say there is nothing that shows the page is loaded. I am all ears on this one. 

Of course a 1 second sleep works, but that doesn't seem right. Myabe we could check for the js ```ready``` ?